build/build-validation.js

```diff
@@ -90,7 +90,6 @@ const schema = {
     ignoreTrailingSlash: { type: 'boolean', default: defaultInitOptions.ignoreTrailingSlash },
     ignoreDuplicateSlashes: { type: 'boolean', default: defaultInitOptions.ignoreDuplicateSlashes },
     disableRequestLogging: {
-      type: 'boolean',
```

Comment:
there seem to be no existing precedents of defining a config value as a function, neither for adding a custom validation keyword, so I went for relaxing the validation altogether.

If a different approach is preferred, I would appreciate being pointed in the right direction.

Reply:
Actually, there is a precedent - `genReqId`, and we do not validate it, so current solution is actually consistent with existing implementation. If we choose to add validation, I'd suggest doing it in a separate PR, and cover both cases.